backend/data/src/main/kotlin/io/tolgee/service/export/ExportFilePathProvider.kt

```diff
@@ -60,19 +53,21 @@ class ExportFilePathProvider(
   }
 
   private fun validateTemplate() {
-    val containsLanguageTag =
-      arrayOf(
-        ExportFilePathPlaceholder.LANGUAGE_TAG,
-        ExportFilePathPlaceholder.ANDROID_LANGUAGE_TAG,
-        ExportFilePathPlaceholder.SNAKE_LANGUAGE_TAG,
-      ).any { getTemplate().contains(it.placeholder) }
-
-    if (!containsLanguageTag) {
-      throw getMissingPlaceholderException(
-        ExportFilePathPlaceholder.LANGUAGE_TAG,
-        ExportFilePathPlaceholder.ANDROID_LANGUAGE_TAG,
-        ExportFilePathPlaceholder.SNAKE_LANGUAGE_TAG,
-      )
+    if (params.format !in listOf(ExportFormat.APPLE_XCSTRINGS)) {
```

Comment:
This is incorrect modifying of file which belongs to different layer. I understand you need to bypass this check for the the xcstrings. 

You can add new boolean property `multiLanguage` to the `ExportFormat` and check for it here.